rs/nns/governance/feature_flags.bzl

```diff
@@ -5,6 +5,6 @@ Feature flags for turning TLA-based checks on and off.
 def test_with_tla():
     return select({
         # Add "tla" here to enable TLA-related checks in the CI
```

Comment:
```suggestion
        # Having "tla" here means that TLA-related checks are enabled in the CI
```

Reply:
Ah sorry, didn't see the comment, already changed it :)